src/System.IO.FileSystem/src/System/IO/Win32FileSystem.cs

```diff
@@ -389,16 +387,12 @@ private static SafeFileHandle OpenHandle(string fullPath, bool asDirectory)
                 throw new ArgumentException(SR.Arg_PathIsVolume, "path");
             }
 
-            Interop.Kernel32.SECURITY_ATTRIBUTES secAttrs = default(Interop.Kernel32.SECURITY_ATTRIBUTES);
-            SafeFileHandle handle = Interop.Kernel32.SafeCreateFile(
+            SafeFileHandle handle = Interop.Kernel32.CreateFile(
```

Comment:
SafeCreateFile was checking for FILE_TYPE_DISK. Can we assume that here?

Reply:
I don't think we can. Why don't we need this? Was it just for CAS?

Reply:
We don't need the check here- it was for creating FileStream, which is no longer in this repo. That said I don't intend to keep it for FileStream either. It goes back to trying to enforce limited trust- and we do anything similar for Unix.